traveltimepy/requests/time_filter_proto.py

```diff
@@ -0,0 +1,197 @@
+import math
+from dataclasses import dataclass
+from enum import Enum
+from typing import ClassVar, Optional, List, Union
+
+import RequestsCommon_pb2  # type: ignore
+import TimeFilterFastRequest_pb2  # type: ignore
+from traveltimepy.requests.common import Coordinates
+
+
+class RequestType(Enum):
+    # single departure location and multiple arrival locations
+    ONE_TO_MANY = "one_to_many"
+    # single arrival location and multiple departure locations
+    MANY_TO_ONE = "many_to_one"
+
+
+@dataclass
+class TransportationInfo:
+    code: int
+    name: str
+
+
+class ProtoTransportation(Enum):
+    PUBLIC_TRANSPORT = TransportationInfo(0, "pt")
+    DRIVING = TransportationInfo(1, "driving")
+    DRIVING_AND_PUBLIC_TRANSPORT = TransportationInfo(2, "pt")
```

Comment:
Should this string be different than "pt"?

Reply:
It is used when creating specific URL, this is correct